genienlp/tasks/almond/__init__.py

```diff
@@ -166,6 +171,10 @@ def preprocess_field(self, sentence, field_name=None):
         new_sentence = ' '.join(tokens)
 
         if self._almond_detokenize_sentence:
+    
+            new_sentence = detokenize_cjk_chars(new_sentence)
+            tokens = new_sentence.split(' ')
```

Comment:
How is this code different that the previous code that detokenizes token-by-token?

Reply:
The previous code wasn't adding spaces after non cjk characters. e.g. "平 巷 8 号 里" became "平巷 8号里" instead of "平巷 8 号里" (with space after 8)
I also changed run_generation to use `tokenize_cjk_chars` and `detokenize_cjk_chars` for consistency.